### ISSUE TITLE
Wrong lines order for NPC entering

### ISSUE THREAD

**Issue by vadi2** [oestrich/ex_venture#56]
```
[101/101hp 66/66sp 24/24mv 25xp]  
You are being targeted by Goblin.

You are now targeting Goblin.

Goblin enters from the west.

[98/101hp 66/66sp 24/24mv 25xp]  
Goblin fails a club at you.
3 bludgeoning damage is dealt to Vadi.
```

**Comment by oestrich** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [NPCs target you before they enter the room](https://trello.com/c/BJr1ZBR2/230-npcs-target-you-before-they-enter-the-room)

**Comment by oestrich** [comment]
Fixed in 4ce150bbe870b53c10c01cab4689668263483590
